client/src/includes/chooserModal.js

```diff
@@ -1,8 +1,10 @@
 /* eslint-disable max-classes-per-file */
 import $ from 'jquery';
-import { initTabs } from './tabs';
-import { initTooltips } from './initTooltips';
-import { gettext } from '../utils/gettext';
+import {initTabs} from './tabs';
+import {initTooltips} from './initTooltips';
+import initChooserCreationForm from './initChooserCreationForm';
+import initChooserFilters from "./chooserFilters";
```

Comment:
Wouldn't it be neater to make `initChooserCreationForm` and `initChooserFilters` into methods on `ChooserModalOnloadHandlerFactory`, so that subclasses of `ChooserModalOnloadHandlerFactory` can customise them where appropriate?

Reply:
Thanks for reviewing this 😃 Thats a great idea. It definitely would be a lot neater to have them inside of the `ChooserModalOnloadHandlerFactory` Chooser creation forms/toggling don't exist outside of the choosers so we definitely wouldn't need them on page load then either yep. 

I agree on removing the `ajaxify-chooser-links` event because we won't need it then either. I'll add this to my TODO list